dipper/sources/ZFIN.py

```diff
@@ -69,8 +71,8 @@ class ZFIN(Source):
             'file': 'zfinpubs.txt',
             'url': ZFDL + '/zfinpubs.txt'},
         'zpmap': {
-            'file': 'zp-mapping.txt',
-            'url': 'http://compbio.charite.de/hudson/job/zp-owl-new/lastSuccessfulBuild/artifact/zp.annot_sourceinfo'},
+            'file': 'zp-mapping-2019.txt',
+            'url': ZPCURATION + '/id_map_zfin.tsv'}, # this is a zp mapping file generated by Nico
```

Comment:
two spaces before an inline comment
but this one exceeds 88 col so move to proceeding line.

Reply:
fixed